suite-native/atoms/src/Modal/BottomModalGestureHandler.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { ReactNode } from 'react';
+import { gestureHandlerRootHOC } from 'react-native-gesture-handler';
+
+/**
+ * On Android RNGH does not work by default because modals are not located under React Native Root view in native hierarchy.
+ * To fix that, components need to be wrapped with gestureHandlerRootHOC (it's no-op on iOS and web).
+ * See more details: https://docs.swmansion.com/react-native-gesture-handler/docs/installation/#usage-with-modals-on-android
+ */
+export const BottomModalGestureHandler = gestureHandlerRootHOC(
```

Comment:
Should it really be in a separate file and not BottomModalContainer? 🤔

Reply:
https://github.com/trezor/trezor-suite/pull/5715/commits/78112c300bb72c8426451c86dd16f066efed6145